components/org.wso2.carbon.identity.remotefetch.core.ui/src/main/resources/web/remotefetch-core/add-remotefetch-config.jsp

```diff
@@ -35,20 +35,14 @@
     pageContext.setAttribute("fetchConfigurationJs", null);
     
     if (request.getParameter("id") != null) {
```

Comment:
Make it as String editId = request.getParameter("id"); and then check for null. if (editId!=null).

Calling this  request.getParameter("id") two times is redundant

Reply:
Resolved